src/tests/ftest/dfuse/pil4dfs_fio.yaml

```diff
@@ -0,0 +1,76 @@
+hosts:
+  test_servers: 2
+  test_clients: 1
+
+timeout: 600
+
+server_config:
+  name: daos_server
+  engines_per_host: 2
+  engines:
+    0:
+      pinned_numa_node: 0
+      fabric_iface: ib0
+      fabric_iface_port: 31317
+      log_file: daos_server0.log
+      storage:
+        0:
+          class: dcpm
+          scm_list: ["/dev/pmem0"]
+          scm_mount: /mnt/daos0
+    1:
+      pinned_numa_node: 1
+      fabric_iface: ib1
+      fabric_iface_port: 31417
+      log_file: daos_server1.log
+      storage:
+        0:
+          class: dcpm
+          scm_list: ["/dev/pmem1"]
+          scm_mount: /mnt/daos1
+
+pool:
+  size: 90%
+  control_method: dmg
```

Comment:
FYI this isn't used anymore. We actually need to remove from existing tests
```suggestion
```

Reply:
- [ ] Remove useless yaml configuration item.

Reply:
Fixed with commit 4c600bcacb04f835b9ca971db94bf9dadb11c05f

Reply:
The control_method for container should stay. It's just control_method under pool that is unused